config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -84,6 +84,7 @@ presubmits:
       testgrid-alert-email: release-managers+alerts@kubernetes.io
       testgrid-num-columns-recent: '30'
   - name: pull-release-image-kube-cross
+    cluster: k8s-infra-prow-build-trusted
```

Comment:
change this to `k8s-infra-prow-build`

change L93 to `serviceAccountName: gcb-builder-releng-test` (ref: https://github.com/kubernetes/k8s.io/pull/1626)

Reply:
thanks @spiffxp. updated